hash.c

```diff
@@ -2171,29 +2094,25 @@ rb_hash_lookup(VALUE hash, VALUE key)
 
 /*
  *  call-seq:
- *    hash.fetch(key) -> value
- *    hash.fetch(key, default) -> value
+ *    hash.fetch(key, default_value = self.default) -> value
```

Comment:
I think this is incorrect.  For one, the description below states that the method does not use the values of method `default`, so it couldn't be the default value of the 2nd argument.  I would keep these three call-seq lines separate.

Reply:
Fixed.